src/guide/javascript.md

```diff
@@ -191,34 +207,135 @@ const kp = generateKeyPair({
 })
 ```
 
-A basic client setup requires a Torii configuration and an account ID. This
-allows you to perform basic operations like health or status checks. As
-described above, to use transactions or queries you'll need to have a
-`keyPair` parameter as a part of the `Client` instance definition:
+When you have a key pair, you might create a `Signer` using the key pair :
 
 ```ts
-import { Client } from '@iroha2/client'
+import { KeyPair } from '@iroha2/crypto-core'
+import { Signer } from '@iroha2/client'
+import { AccountId, DomainId } from '@iroha2/data-model'
 
-const client = new Client({
-  torii: {
-    // Both URLs are optional in case you only need one of them,
-    // e.g. only the telemetry endpoints
-    apiURL: 'http://127.0.0.1:8080',
-    telemetryURL: 'http://127.0.0.1:8081',
-  },
-  accountId: AccountId({
-    // Account name
-    name: 'alice',
-    // The domain where this account is registered
-    domain_id: DomainId({
-      name: 'wonderland',
-    }),
+// Key pair from previous step
+declare const keyPair: KeyPair
+
+const accountId = AccountId({
+  // Account name
+  name: 'alice',
+  // The domain where this account is registered
+  domain_id: DomainId({
+    name: 'wonderland',
   }),
-  // A key pair, needed for transactions and queries
-  keyPair: kp,
 })
+
+const signer = new Signer(accountId, keyPair)
+```
+
+Well, now we are able to make signatures with `signer.sign(binary)`! But
+usually we need to interact with Iroha and send something to it, like
+transactions or queries. `Torii` will help us.
+
+`Torii` class represents HTTP / WebSocket communications with Iroha. It
+will help us to communicate with each Iroha Endpoint, i.e.:
+
+- Submit transactions
+- Make queries
+- Listen for events
+- Listen for blocks stream
+- and so on
+
+To initialize `Torii`, first we need to know Iroha Torii URLs. Our Iroha
+Peer is configured to listen for API endpoints at `http://127.0.0.1:8080`
+and for telemetry endpoints at `http://127.0.0.1:8081`. Then, we need to
+provide appropriate HTTP / WebSocket adapters which `Torii` will use[^1].
+These adapters depends on the environment you are going to use
+`@iroha2/client` in.
+
+[^1]:
+    We have to pass environment-specific `ws` and `fetch`, because there is
+    no way for Iroha Client to communicate with a peer in an
+    environment-agnostic way.
+
+In Node.js `Torii` initialization will look like this:
+
+```ts
+import { Torii } from '@iroha2/client'
+import { adapter as WS } from '@iroha2/client/web-socket/node'
+
+import nodeFetch from 'node-fetch'
+
+// another alternative
```

Comment:
if there are different alternatives, I think they should be in different code snippets

Reply:
Oops, forgot to delete it here.

Reply:
Cleaned.